t/50mruby-channel.t

```diff
@@ -0,0 +1,95 @@
+use strict;
+use warnings;
+use Test::More;
+use Test::Exception;
+use t::Util;
+
+plan skip_all => 'mruby support is off'
+    unless server_features()->{mruby};
+
+subtest "process integer" => sub {
+    my $server = spawn_h2o(<< 'EOT');
+hosts:
+  default:
+    paths:
+      /:
+        mruby.handler: |
+          Proc.new do |env|
+            ch = create_channel
+            ch.push 1
+            res = ch.shift
+            [200, {}, [res]]
+          end
+
+EOT
+    my ($stderr, $stdout) = run_prog("curl --silent --dump-header /dev/stderr http://127.0.0.1:$server->{port}/");
+    is $stdout, 1;
+};
+
+plan skip_all => 'mruby support is off'
+    unless server_features()->{mruby};
+
+subtest "process string" => sub {
+    my $server = spawn_h2o(<< 'EOT');
+hosts:
+  default:
+    paths:
+      /:
+        mruby.handler: |
+          Proc.new do |env|
+            ch = create_channel
```

Comment:
H2O::Channel.new?

Reply:
I fix to use new for creating instances.
https://github.com/h2o/h2o/pull/1336/commits/7edafbc7cb5f5e1960075a59a4bb30f2b5efe5b6